requirements.txt

```diff
@@ -22,3 +22,5 @@ tensorflow==2.1.0
 keras==2.3.1
 sacremoses>=0.0.41
 protobuf<=3.20.1
+multiprocess==0.70.12.2
```

Comment:
Why are we pinning this? From the PR it is not immediately obvious. If we really need this, I would add it in setup.cfg and I would relax the requirements if possible.

Reply:
multiprocess updated to `0.70.13` two weeks ago. 
With the new version I had problems using `gt4sd`. 
Downgrading to `0.70.12.2` everything works fine.